### ISSUE TITLE
PDF not show when browser zoom is low or low resolution

### ISSUE THREAD

**Issue by Sarebok85** [VadimDez/ng2-pdf-viewer#299]
- [x] Bug report -> please search issues before submitting

Hi, i have a problem with component. I have a Blob that get URL to use SRC in component and when i down the zoom in the browser (Chrome, for example, but happend in all browser) cant see PDF. I see the LoadingIcon and not appear the PDF.

Html:
`<pdf-viewer [src]="urlPDF" style="display: block;"></pdf-viewer>
`

Component:
```
this.blobPDF = this.b64toBlob([BLOB], "application/pdf");
this.urlPDF = URL.createObjectURL(this.blobPDF);
```

B64toBlob Function:
```
private b64toBlob(b64Data, contentType, sliceSize = 512): Blob {
const byteCharacters = atob(b64Data);
const byteArrays = [];

    for (let offset = 0; offset < byteCharacters.length; offset += sliceSize) {
      const slice = byteCharacters.slice(offset, offset + sliceSize);

      const byteNumbers = new Array(slice.length);
      for (let i = 0; i < slice.length; i++) {
        byteNumbers[i] = slice.charCodeAt(i);
      }

      const byteArray = new Uint8Array(byteNumbers);

      byteArrays.push(byteArray);
    }

    const blob = new Blob(byteArrays, { type: contentType });
    return blob;
  }
```

![captura](https://user-images.githubusercontent.com/24222160/37330372-2169b072-26a1-11e8-8dcc-fedff8757fcd.PNG)

**Comment by waterskier2007** [comment]
@VadimDez is there any update on this?

**Comment by ragafus** [comment]
Same problem happening here with the latest version (5.2.1) and zoom<=50%

**Comment by MilanRaval** [comment]
I have similar issue but for zoom > 0.98. PDF is visible till I set zoom = 0.98 but if zoom is set to 0.99 or more than it shows loading icon.

**Comment by VadimDez** [comment]
Fixed in `6.0.0` See https://github.com/VadimDez/ng2-pdf-viewer/blob/master/CHANGELOG.md#600

**Comment by neetuvdb** [comment]
can anyone tell me the step, I got many errors like ERROR in node_modules/ng2-pdf-viewer/src/app/pdf-viewer/pdf-viewer.module.d.ts:7:100 - error TS2305: Module '"../../../../@types/pdfjs-dist"' has no exported member 'PDFPromise'

I did removed ng_module and  update version like:

"ng2-pdf-viewer": "6.1.3",
    "pdfjs-dist": "2.5.207",

"ng2-pdf-viewer": "~5.3.4"
"pdfjs-dist": "~2.5.207"

not worked.
